# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-backgrounds/animations/background-color-animation-with-zero-alpha.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/css/css-backgrounds/animations/background-color-animation-with-zero-alpha.html.ini
--- a/testing/web-platform/meta/css/css-backgrounds/animations/background-color-animation-with-zero-alpha.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/css/css-backgrounds/animations/background-color-animation-with-zero-alpha.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,9 +1,8 @@
 [background-color-animation-with-zero-alpha.html]
   expected:
-    if webrender and fission and (os == "linux") and not debug: ["TIMEOUT", "PASS"]
-    if webrender and not fission and (os == "android") and debug: PASS
     if not webrender and (os == "android") and debug: PASS
+    if webrender and (os == "mac"): ["TIMEOUT", "PASS"]
+    if not webrender and (os == "win"): PASS
     if not webrender and (os == "linux"): PASS
     if not webrender and (os == "mac"): PASS
-    if not webrender and (os == "win"): PASS
     [PASS, TIMEOUT]